internal/iam/repository_role_grant_scope_test.go

```diff
@@ -670,7 +670,7 @@ func Test_SetRoleGrantScope(t *testing.T) {
 			expectRoleVersionChange: 0,
 			scopes:                  []string{globals.GrantScopeChildren, globals.GrantScopeDescendants},
 			wantErr:                 true,
-			wantErrMsg:              "iam.(Repository).SetRoleGrantScopes: only one of ['children', 'descendants'] can be specified: parameter violation: error #100",
```

Comment:
This error might've been clearer before. I personally think I prefer `'` to ` 😅 . I won't insist on it though.